lib/msf/core/payload/android.rb

```diff
@@ -9,6 +9,8 @@ module Msf::Payload::Android
   include Msf::Payload::TransportConfig
   include Msf::Payload::UUID::Options
 
+  @@jar_signing_key = OpenSSL::PKey::RSA.new(2048)
```

Comment:
Thoughts on making this lazy instantiated? i.e. Loading this module doesn't create a key, but the first access will:

```ruby
def signing_key
  @@signing_key ||= OpenSSL::PKey::RSA.new(2048)
end
```

Reply:
Looks good, will add it in 👍